public/app/plugins/panel/canvas/editor/options.ts

```diff
@@ -120,4 +126,21 @@ export const optionBuilder: OptionSuppliers = {
       },
     });
   },
+
+  addType: (builder, context) => {
+    const category = ['Type'];
+    builder.addSelect({
+      category,
+      // id: 'type',
```

Comment:
Should this be commented out?

Reply:
We don't need it because `id` property created inside `addSelect` method from `path` property.

https://github.com/grafana/grafana/blob/main/packages/grafana-data/src/utils/OptionsUIBuilders.ts#L283